sdk/communication/azure-communication-rooms/src/main/java/com/azure/communication/rooms/RoomsServiceVersion.java

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.communication.rooms;
+
+import com.azure.core.util.ServiceVersion;
+
+/**
+ * The versions of Communication Rooms Service supported by this client library.
+ */
+public enum RoomsServiceVersion implements ServiceVersion {
+    /**
+     * Service version {@code 2021-04-07}
+     */
+    V2021_04_07_PREVIEW("2021-04-07"),
```

Comment:
Doesn't the service version have "preview" suffix?

Reply:
Addressed.